### ISSUE TITLE
YT doesn't make use of any queue system?

### ISSUE THREAD

**Issue by mauzao9** [flyth/ts3soundbot#273]
Hello there.

I have been testing new version and it seems there is no queue at all on the yt commands, that on our ts are pretty much the whole way music is played.

Do you know if it's intended or you have plans to integrate this command into the queue functionality you just implemented?

Would be much better, giving the playback permission everybody can overlap each other sadly.

**Comment by flyth** [comment]
There's no queue version of !yt right now, yup. Will add that. :)

**Comment by supimfuzzy** [comment]
Direct Youtube search integration would also be a good addition. Something like what www.watchtogether.com does.

**Comment by GavoTrav** [comment]
any update on this @flyth

**Comment by flyth** [comment]
Yes, please use the most recent beta from the forums. It is capable of importing playlists from ytdl.

**Comment by Wilbo007** [comment]
u lazy fuck get working on it
